ui/app/AppLayouts/Chat/CommunityComponents/CommunityProfilePopupMembersList.qml

```diff
@@ -39,14 +40,20 @@ Item {
         anchors.rightMargin: -Style.current.padding
     }
 
-    MembershipRequestsButton {
```

Comment:
Can we remove this component then?

Reply:
It seems like this component is still used in the left pane of the community view